packages/cli/src/commands/start.ts

```diff
@@ -198,6 +198,42 @@ async function askQuestions({ template: defaultTemplate }: Pick<StartOptions, "t
       process.exit(0);
     }
     context.services = services;
+
+    // Ask for environment variables
+    const envList = Array.from(new Set(connectors.filter(c => services.includes(c.serviceName)).flatMap((c) => {
+      return c.envList;
+    })));
+
+    const isConfirm = await p.confirm({
+      message: `Do you want to enter environment variables? (Number of environment variables to enter: about ${envList.length})`,
+      initialValue: false,
+    });
+
+    const envInfos: EnvInfo[] = [];
+
+    if (p.isCancel(isConfirm) || !isConfirm) {
+      p.cancel("Skipping environment variables input.");
+    }
+    else {
+      for (const env of envList) {
+        const envValue = await p.text({
+          message: `${env}: `,
+          defaultValue: "",
+        });
+
+        if (p.isCancel(envValue)) {
+          p.cancel("Skipping environment variables input.");
+          break;
```

Comment:
Skiped values must be filled with empty values.

Reply:
![image](https://github.com/user-attachments/assets/60dde2e6-a7de-48ae-8904-bf7c611bf716)

like this?

Reply:
yeah good
if user also cancel type env, but user should know to need what kind of env variables